### DISCUSSION TITLE
Is it possible to control the GUI cursor?

### DISCUSSION THREAD

**Discussion by fuzzy7k** [AntiMicroX/antimicrox#747] [Q&A] [answered]
I would like to do general GUI navigation using a controller. I have done this in Xorg, but things have changed since then. I have antimicrox installed via apt, and can load the program to set bindings. Button mapping works within the antimicrox application itself, but not anywhere else. I have the right stick set to control the mouse. Unsure of how to activate the profile, I tried loading it via the command line with `antimicrox --no-tray --profile ~/Templates/xbox360.gamecontroller.amgp --profile-controller xbox360 --startSet 1 --daemon`
Unfortunately, the joy does not move the cursor. If I try to set a profile autoload for something like firefox, "Detect Window Properties" does not work when clicking the window. It appears to only detect qt based applications. So, I assume this is an out of scope issue. 

Before digging to deeply into this, I am curious if this should work under ideal conditions, with a pure Qt environment.

**Comment by pktiuk** [comment]
I don't recommend using apt packages, they are not maintained anymore. Fedora, suse, arch are fine, but debian ones are not maintained anymore.

**Reply by fuzzy7k** [reply]
Yeah, I noticed that it segfaults when run in daemon mode.

**Comment by fuzzy7k** [comment, marked as answer]
Ok, I figured this out. I needed to use the --eventgen uinput option, plus, modify the permission on /dev/uinput to be able to write to it.
